### PR TITLE
docs(community): landing page

### PR DESCRIPTION
**Pull request by skim1420**
@spinnaker/reviewers closed the other one because i can't git

### CODE REVIEW

File: community/index.md [hunk 1]
```diff
@@ -5,9 +5,27 @@ sidebar:
...
+* [StackOverflow](https://stackoverflow.com/search?q=spinnaker)
+
+## Core Contributors
+
+The list of contributors is growing with some core contributors below. Please reach out to us with a PR or on slack if you'd like to be added to this list!
+
+* [Cerner Corporation](http://cerner.com/)
+* [Google](https://opensource.google.com/)
+* [Kenzan](http://kenzan.com/)
+* [Microsoft/Azure Devops](https://github.com/azure-devops)
```
Comment (lwander) [inline community/index.md:23]: Why are veritas & azure github links? Also why no link for target?

Comment (skim1420) [reply community/index.md:23]: That's the link they requested. I don't have a link for Target (target.com is the retail website, not a corporate site)

Comment (lwander) [reply community/index.md:23]: Sounds good

**Comment by ttomsu** [comment]
LGTM

**Pull request merged**